src/runtime_src/core/tools/xbutil2/SubCmdJsonObjects.cpp

```diff
@@ -0,0 +1,153 @@
+#include <iostream>
```

Comment:
This file is missing copyright and for sure many includes.  Please include-what-you-use, e.g. do no rely on indirect includes and never include more than what you use.